src/battle_script_commands.c

```diff
@@ -2168,7 +2170,7 @@ static void Cmd_printselectionstringfromtable(void)
 {
     if (gBattleControllerExecFlags == 0)
     {
-        const u16 *ptr = (const u16*) T1_READ_PTR(gBattlescriptCurrInstr + 1);
+        const u16* ptr = (const u16*) T1_READ_PTR(gBattlescriptCurrInstr + 1);
```

Comment:
Doesn't the repo tend to prefer `const u16 *` to `const u16* `?

Reply:
> Doesn't the repo tend to prefer `const u16 *` to `const u16* `?

Is this the case for both pokeemerald and pokefirered? If that's the case I can change all pointers so that they're `uX *` instead of `uX*` in both of the repos

Reply:
The star on the left is more common in C++, almost every C codebase I can think of puts the star on the right :)